apps/server/apis/grpcapi/external.go

```diff
@@ -530,6 +530,26 @@ func (s *ExternalServer) AttachPipeline(ctx context.Context, req *protos.AttachP
 		return util.StandardResponse(ctx, protos.ResponseCode_RESPONSE_CODE_INTERNAL_SERVER_ERROR, err.Error()), nil
 	}
 
+	// We need to know if this audience has any other pipelines already attached.
+	// If no, we will need to inject infer schema. If not, infer schema is probably
+	// already there.
+	//
+	// <<<<<<<PROBLEM>>>>>>>>
+	//
+	// TODO: We cannot send directly via channel - this will only work for one
+	// node. We need to broadcast to all nodes but broadcast expects
+	// AttachPipelineRequest which only specifies pipeline id.
+	// Meaning, to make sure this works properly, we need to create an "invisible"
```

Comment:
Maybe we could handle on the broadcast consumer.
That way the individual server is responsible for attaching the ephemeral pipeline if it has connected clients